pages/code-of-conduct.js

```diff
@@ -0,0 +1,23 @@
+import Link from "next/link";
+
+//import Footer from "../components/footer"
```

Comment:
don't forget to delete the commented code :eyes:

Reply:
This file has been deleted, because it does not belong to the branch, and does not have the changes for the page view